eshell-toggle.el

```diff
@@ -74,6 +74,13 @@
                  (const :tag "Enabled" t))
   :group 'eshell-toggle)
 
+(defcustom eshell-toggle-use-project-root
```

Comment:
probably we should just add another option to `eshell-toggle-use-projectile-root`, smth like `nil` for "Disabled", `t` and `'projectile` for projectile, and `'project` for using project.el. And